### PR TITLE
LPS-162319 Poshi Automation for Bug | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-162319

cc @david-gutierrez-mesa 
cc @Tim-Cao

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsConfiguration.testcase [hunk 1]
```diff
@@ -47,6 +47,130 @@ definition {
...
+				FragmentsAdminNavigator.gotoCollection(collectionName = "Collection Name");
+
+				FragmentsAdminNavigator.gotoEntryEllipsisItem(
+					entry = "Fragment Name",
+					item = "Edit");
+
+				var htmlSnippet = '''
+<div class="fragment-name">
+	<img
+		data-lfr-editable-id="img-background"
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsConfiguration.testcase:119]: The same as above

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsConfiguration.testcase [hunk 2]
```diff
@@ -47,6 +47,130 @@ definition {
...
+		data-lfr-editable-type="image"
+		class="media-img"
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsConfiguration.testcase:121]: The same as above

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsConfiguration.testcase [hunk 3]
```diff
@@ -47,6 +47,130 @@ definition {
...
+				imageFileName = "document_2-jpg");
+
+			PageEditor.publish();
+		}
+
+		task ("Then the page creator should see the image on content page at view mode") {
+			ContentPagesNavigator.openViewContentPage(
+				pageName = "Test Page Name",
+				siteName = "Test Site Name");
+
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsConfiguration.testcase:165]: Use ContentPages.viewFragmentImage

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsConfiguration.testcase [hunk 4]
```diff
@@ -47,6 +47,130 @@ definition {
...
+				FragmentsAdmin.propagateChanges();
+			}
+		}
+
+		task ("When the page creator publishes the page after change resource image") {
+			ContentPagesNavigator.openEditContentPage(
+				pageName = "Test Page Name",
+				siteName = "Test Site Name");
+
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsConfiguration.testcase:147]: Add comment to mention missing step until https://issues.liferay.com/browse/LPS-158453 fixed

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, would you mind updating according to comments? Thanks

**Comment by ozysouza** [comment]
> Hi @ozysouza, would you mind updating according to comments? Thanks

Hi @Tim-Cao, PR updated! Could you please have a look at? Thanks!

**Comment by Tim-Cao** [comment]
LGTM!

**Comment by Tim-Cao** [comment]
ci:forward

**Pull request closed without merging**